### DISCUSSION TITLE
Browser addons login network error with NGINX proxy and IPv6

### DISCUSSION THREAD

**Discussion by mgielissen** [dani-garcia/vaultwarden#1394] [Q&A] [answered]
Does the Bitwarden addon in Firefox and Chrome support IPv6? I have a NGINX proxy with IPv6 enabled. Logging in with the addons gives an network error. With IPv4 it works fine.

**Comment by mgielissen** [comment, marked as answer]
Support for IPv6 in NGINX needed?

listen [::]:443 ssl http2;

**Comment by nulano** [comment]
Works fine for me over IPv6 and NGINX proxy.

**Reply by mgielissen** [reply]
I have multiple virtual hosts on IPv6, adding **listen [::]:443 ssl http2;** seems to solved it. Now it works fine.
